server.js

```diff
@@ -41,19 +41,27 @@ const checkCredentials = async () => {
         throw new Error(error);
     }
 };
+
+const checkCampaign = async () => {
+    try {
+        await client.campaigns.get("Reward Promotion");
+    } catch (error) {
+        if (error.code === 404) {
+                const msg = "The 'Reward Promotion' campaign not found. \r\nPlease run the command 'node ./add-missing-campaign.js' in the terminal to create the missing campaign. \r\nWhen you receive a message that the campaign has been successfully created you can run the main script again. \r\nYou can get additional support here: https://github.com/voucherifyio/voucherify-examples/tree/main#get-support- \r\n";
```

Comment:
It's a little hard to read it. Also, \r\n is Windows-specific. Maybe:

```ts
const { EOL } = require('os');
 // ...
const msgs = [
  'The \'Reward Promotion\' campaign not found',
  'Please run the command \'node ./add-missing-campaign.js\' in the terminal to create the missing campaign.',
  'When you receive a message that the campaign has been successfully created you can run the main script again.',
  'You can get additional support here: https://github.com/voucherifyio/voucherify-examples/tree/main#get-support'
]

// try
console.log(msgs.join(EOL))
// or
console.log(...msgs)
```

Reply:
Thanks for the suggestion to use EOL! :) 🤦🏽‍♀️

Reply:
changes have been made